### PR TITLE
VS-695. Updates to run Precision and Sensitivity on VQSR Lite

### PR DESCRIPTION
**Pull request by gbggrant**
Update GvsCalculatePrecisionAndSensitivity.wdl to allow for different scale of calibration_sensitivity vs. lod score.
Also retrieving score from JointVcfFiltering and storing that in BQ and in the VCF. (Probably don't need this long term.)

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java [hunk 1]
```diff
@@ -141,14 +141,14 @@ public enum SensitivityFilteringType { GENOTYPE, SITES, NONE }
 
     @Argument(
             fullName ="snps-truth-sensitivity-filter-level",
-            doc = "The truth sensitivity level at which to start filtering SNPs",
+            doc = "The truth sensitivity level above which to start filtering SNPs",
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java:144]: isn't it...below?

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java [hunk 2]
```diff
@@ -247,6 +248,12 @@ protected String[] customCommandLineValidation() {
                 errors.add("Parameters 'project-id', 'dataset-id', 'call-set-identifier', 'wdl-step', 'wdl-call', and 'shardIdentifier' must be set if 'cost-observability-tablename' is set.");
             }
         }
+        if (truthSensitivitySNPThreshold < 0.0 || truthSensitivitySNPThreshold > 1.0) {
+            errors.add("Parameter 'snps-truth-sensitivity-filter-level' MUST be between 0.0 and 1.0'");
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java:252]: should we print out what the number actually is?

**Review by RoriCremer — APPROVED** [review]

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateFilteringFiles.java [hunk 3]
```diff
@@ -37,10 +37,10 @@ public final class CreateFilteringFiles extends VariantWalker {
     private SimpleXSVWriter writer;
 
     private List<String> HEADER = 
-        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "vqslod", "culprit","training_label","yng");
+        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "vqslod", "culprit", "training_label", "yng");
     // TODO - not sure about culprit and training label.
     private List<String> HEADER_VQSR_LITE =
-        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "calibration_sensitivity", "culprit", "training_label", "yng");
+        Arrays.asList("filter_set_name", "mode", "location", "ref", "alt", "calibration_sensitivity", "score", "training_label", "yng");
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateFilteringFiles.java:43]: oh thank god--"culprit" is so confusing

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java (hunk 1 above)
Comment (gbggrant) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java:144]: Actually no. For VQS_LOD it was below, but for calibration_sensitivity it's flipped - so we filter gts/sites which have calibration_sensitivity > the truth-sensitivity-filter-level

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateFilteringFiles.java (hunk 3 above)
Comment (gbggrant) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateFilteringFiles.java:43]: Actually, it's gone in LITE - there isn't even a culprit provided. The 'score' is analogous to VQS_LOD (as generated by VQSR classic)

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java (hunk 2 above)
Comment (gbggrant) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java]: Done

**Review by mcovarr — APPROVED** [review]
LGTM

**Pull request merged**